package.json

```
@@ -0,0 +1,27 @@
+{
+  "devDependencies": {
+    "tailwindcss": "^4.1.4"
+  },
+  "name": "jofas.github.io",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "tailwindcss --watch"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Vis-halV/jofas.github.io.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Vis-halV/jofas.github.io/issues"
+  },
+  "homepage": "https://github.com/Vis-halV/jofas.github.io#readme",
+  "dependencies": {
```

For now, we only need the dependencies in the `package.json`:

```suggestion
    "tailwindcss": "^4.1.4",
    "@tailwindcss/cli": "^4.1.4"
```